lib/ui/surveydetail/survey_question_page.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/gen/assets.gen.dart';
+import 'package:flutter_survey/model/question.dart';
+import 'package:flutter_survey/ui/surveydetail/survey_detail_screen.dart';
+import 'package:flutter_survey/ui/surveydetail/survey_question_content.dart';
+
+class SurveyQuestionPage extends StatelessWidget {
+  final Question question;
+  final int index;
+  final int total;
+  final Function() onClose;
+
+  const SurveyQuestionPage({
+    super.key,
+    required this.question,
+    required this.index,
+    required this.total,
+    required this.onClose,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return _buildPage(context);
+  }
+
+  Column _buildPage(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          children: [
+            const Expanded(child: SizedBox.shrink()),
+            Padding(
+              padding: const EdgeInsets.only(left: 20),
+              child: GestureDetector(
```

Comment:
This one we can use `IconButton` so we can get rid of `GestureDetector`

Reply:
Resolved in https://github.com/nimblehq/ic-flutter-avishek/pull/58/commits/c3e28b47d8e45884f706677f35777c7eb264e6f0